bedrock/firefox/views.py

```diff
@@ -370,6 +380,15 @@ def show_36_whatsnew_tour(oldversion):
     return oldversion < Version('36.0')
 
 
+def show_38_0_5_firstrun(version):
+    try:
+        version = Version(version)
+    except ValueError:
+        return False
+
+    return version >= Version('38.0.5') and version < Version('39.0')
```

Comment:
Is `and version < Version('39.0')` needed here - is this new content not be what we're showing people going forward, or until we next update /firstrun?